samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -57,9 +57,19 @@ class InferReqWrap final {
 
     void infer() {
         _startTime = Time::now();
-        _request.infer();
+        _request.Infer();
```

Comment:
please double check the code. ov::runtime::InferRequest has 'infer' method, not 'Infer'